plugins/hydra_ray_launcher/tools/ray_cluster.yaml

```diff
@@ -0,0 +1,51 @@
+cluster_name: ray_test_base_AMI
```

Comment:
this looks familiar.
didn't I see it as a hard coded string somewhere?

Reply:
fixed by making the script a Hydra app 💪